tigerpath/majors_and_certificates/majors/ECO.json

```diff
@@ -185,6 +188,7 @@
           "course_list": [
             "COS 445",
             "ELE 381",
+            "ENE 422/MAE 422",

```

Comment:
https://economics.princeton.edu/undergraduate-program/course-selection/pre-approved-cognates/

This page applies to all changes below.

Reply:
I think we're missing POL 432 and POL 505.

Reply:
Gotcha , fixed.